plugin/vim-carbon-now-sh.vim

```diff
@@ -4,18 +4,18 @@ endif
 
 let g:carbon_now_sh_loaded = 1
 
-let g:carbon_now_sh_options = get(g:, 'carbon_now_sh_options', 't=material')
+let g:carbon_now_sh_options = get(g:, 'carbon_now_sh_options', '{}')
 let g:carbon_now_sh_browser = get(g:, 'carbon_now_sh_browser', '')
 
 command! -range=% CarbonNowSh <line1>,<line2>call s:carbonNowSh()
 
 function! s:carbonNowSh() range
-  let l:text = s:getVisualSelection()
+  let l:text = s:urlEncode(s:getVisualSelection())
   let l:browser = s:getBrowser()
-  let l:options = escape(g:carbon_now_sh_options, '&')
+  let l:options = type(g:carbon_now_sh_options) == v:t_dict ? s:getOptions() : g:carbon_now_sh_options
```

Comment:
I've added type checking here so that if `options` is set as a string, things should work just as before.